sklearn/model_selection/_split.py

```diff
@@ -1006,6 +1006,13 @@ def _iter_indices(self, X, y, labels):
             yield train, test
 
 
+def _sample_without_replacement(class_counts, n_samples, rng):
+    n_classes = len(class_counts)
```

Comment:
we could but I'd rather make the logic of the function simple. What happens if you pass the wrong length?